packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/WebhookForm/utils/makeWebhookValidationSchema.js

```diff
@@ -14,7 +14,13 @@ export const makeWebhookValidationSchema = ({ formatMessage }) =>
           defaultMessage: 'Name is required',
         })
       )
-      .matches(NAME_REGEX, translatedErrors.regex),
+      .matches(
+        NAME_REGEX,
+        formatMessage({
+          id: translatedErrors.regex,
+          defaultMessage: 'The value does not match the regex',
```

Comment:
Are we able to describe the regex in a human readable way? e.g. "contain no special characters" (I've not read the regex)

Reply:
@joshuaellis wdyt of the error messaging in [`653d1e0` (#17103)](https://github.com/strapi/strapi/pull/17103/commits/653d1e0fd719bc028ab45d81b6816c57acaed828) ?